sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_file_client_async.py

```diff
@@ -144,13 +143,14 @@ def __init__(  # type: ignore
         # type: (...) -> None
         kwargs["retry_policy"] = kwargs.get("retry_policy") or ExponentialRetry(**kwargs)
         loop = kwargs.pop('loop', None)
+        if loop:
+            warnings.warn("``loop`` is no longer supported.", DeprecationWarning)
```

Comment:
We could more explicitly check for Python > 3.10. Also I think we can update the message to align with the [official Python statement](https://docs.python.org/3/whatsnew/3.10.html#whatsnew310-removed):
```python
if loop and sys.version_info >= (3, 8):
    warnings.warn("The 'loop' parameter was deprecated from asyncio's high-level APIs in Python 3.8 and is no longer supported.", DeprecationWarning)
```
Lastly as mentioned below - if we were actually using this anywhere - we'll need to upgrade from a warning to a ValueError.

Reply:
See comment on other issue. Resolved 😄